build/Rakefile

```diff
@@ -37,9 +37,23 @@ namespace :cluster do
     puts "Deploy cluster config: #{path(args)}"
     system("cd #{path(args)} && terraform apply") or exit 1
     puts "Cluster #{path(args)} created successfully"
+
+    puts "Deploy the EBS CSI plugin on k8s"
+    cmd = %Q{kubectl apply -k "github.com/kubernetes-sigs/aws-ebs-csi-driver/deploy/kubernetes/overlays/stable/?ref=master"}
```

Comment:
Hey @ilchebedelovski what's this and why is it needed? Also this might not be the best place for it as we want these rake tasks to support gcp, az,etc in the furture too.

Reply:
@utx0 ideally, this should be placed in the Terraform module, this is the CSI plugin deployment from here https://aws.amazon.com/premiumsupport/knowledge-center/eks-persistent-storage/ , but that `-k` option is not supported for kubectl in Terraform, that's why I have placed in the Rakefile.

Maybe we can place it in sifnode:install function in Rakefile, what do you think?

Reply:
I would prefer this is added to the aws terraform modules some how. Any reason that cant happen?

Reply:
It is because we cannot execute `kubectl apply -k` over the kubectl provider in Terraform, this is important because we are installing whole directory and -k means `-k, --kustomize='': Process a kustomization directory.`.

Reply:
Would this help: https://github.com/kbst/terraform-provider-kustomization

Reply:
@utx0 I have solved the issue with the kustomization provider, thanks